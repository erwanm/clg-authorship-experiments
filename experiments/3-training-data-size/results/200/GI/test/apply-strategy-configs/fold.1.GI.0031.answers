1886-hj-pc 1897-cdw-tpfwsw	0.599696
1852-hbs-utc 1896-hj-toh	0.562992
1896-espw-tsasa 1905-fmc-fm	0.536564
1883-fmc-ars 1911-hj-to	0.619516
1882-wta-aa 1898-fmc-ari-v2	0.55198
1877-hj-ta 1880-hj-ws	0.664296
1893-fmc-md 1905-fmc-wso	0.442601
1887-fmc-s 1891-fmc-k	0.747981
1885-wta-sts 1894-cdw-tgh	0.58061
1871-haj-tt 1883-haj-tbb	0.422781
1852-hbs-utc 1871-hbs-pawt	0.579142
1895-cfw-mcac 1906-mt-cfma	0.435995
1866-wta-bos 1876-wta-vao	0.53216
1864-lma-m 1904-rwc-sm	0.7656
1886-hj-tb 1900-espw-wtg	0.72082
1881-mt-tpatp 1900-espw-wtg	0.426451
1872-haj-ptf 1898-haj-abf	0.844147
1882-fmc-mi 1908-fmc-tws	0.658423
1898-hj-itc 1901-hj-tsf	0.502062
1871-espw-tsp 1907-espw-tldup	0.432324
1871-espw-tsp 1911-rwc-tcl	0.430122
1866-wta-ob 1883-haj-tbb	0.537299
1871-hbs-pawt 1880-hj-c	0.644475
1872-haj-ptf 1893-haj-dtn	0.592355
1896-mt-projoa 1907-ab-rm	0.525553
1888-es-ttatv 1893-fmc-kl-v2	0.831667
1897-ab-tdohy 1911-us-alp	0.894799
1885-fmc-zamc 1887-fmc-s	0.58061
1906-mt-cfma 1915-rwc-b	0.366256
1885-fmc-tcotk 1900-fmc-itpotk	0.78542
1883-fmc-dc 1890-fmc-acmr	0.575471
1906-fmc-alor 1908-fmc-tp	1
1889-fmc-g 1893-fmc-md	0.819188
1857-wta-ta 1895-ab-mgtonel	0.41544
1883-hj-rh 1890-hj-ttm	0.521149
1883-fmc-tl 1886-hj-pc	0.684116
1895-fmc-t 1910-us-sts	0.475635
1897-cdw-tpfwsw 1922-rwc-tfj	0.251739
1898-hj-ttm 1917-ab-bn	0.41177
1864-lma-opdaot 1897-ewaoc-tdof	0.457283
1882-haj-ffbts 1886-hbs-tsw	0.892597
1895-wta-alae 1899-hj-taa	0.471965
1871-haj-tt 1878-hbs-pp	0.372129
1908-us-tm 1921-us-tbol	0.524085
1854-hbs-smofl-v2 1896-mt-projoa	0.56446
1881-hj-tpoal 1903-hj-ta	0.485912
1893-fmc-md 1906-mt-cfma	0.349372
1901-espw-a 1904-espw-t	0.240728
1882-wta-aa 1890-wta-otb	1
1880-hj-c 1896-hj-toh	0.947653
1858-wta-pap 1870-wta-faf	0.602632
1891-fmc-do 1894-fmc-wg	0.656221
1868-lma-lw 1898-haj-abf	0.634198
1884-wta-Sac 1910-us-sts	0.565928
1881-hj-tpoal 1885-hj-gr	0.813315
1888-fmc-si 1894-fmc-wg	0.78909
1905-fmc-fm 1908-fmc-tws	0.781749
1866-haj-hf 1916-mt-tms	0.903608
1880-hj-ws 1885-hj-altof	0.362586
1914-us-sm 1921-us-tbol	0.348638
1903-us-tjoas 1908-us-tm	0.43159
1878-hj-te 1902-hj-twotd	0.457283
1866-haj-hf 1893-haj-ianwatgfoa	0.71715
1885-hj-altof 1888-hj-tap	0.480774
1895-fmc-ajs 1898-fmc-ari-v2	0.225312
1874-haj-rftr 1922-us-tcmc	0.405897
1869-wta-dtr 1910-ew-tomag	0.342766
1855-hbs-tmf 1883-fmc-tl	0.759727
1880-hj-ws 1894-espw-asl	0.571801
1864-lma-opdaot 1883-fmc-tl	0.673839
1858-wta-pap 1874-wta-tcw	1
1880-hj-ws 1913-ew-tcotc	0.672371
1859-hbs-tmw 1862-hbs-tpooi	0.934439
1871-haj-tt 1891-haj-tetb	0.409567
1886-hj-tb 1904-hj-tgb	0.623921
1896-hj-tsop 1918-rwc-tlg	0.802304
1866-haj-hf 1898-haj-abf	0.676775
1867-espw-gyatgc 1900-espw-wtg	0.460219
1893-fmc-kl-v2 1893-fmc-tr	0.773674
1895-espw-tsojc 1898-fmc-vc	0.311934
1883-fmc-dc 1893-fmc-md	0.796431
1896-espw-cfal 1905-haj-fftf	0.344968
1898-hj-itc 1921-us-tbol	0.397822
1903-us-og 1918-us-jh	0.831667
1874-wta-tcw 1880-wta-ds	0.710543
1876-haj-btlb 1892-haj-dfg	0.524085
1878-hj-aie 1909-rwc-tdm	0.560055
1902-fmc-c 1916-ab-tp	0.443335
1883-hj-rh-v2 1897-hj-wmk	0.413238
1895-wta-alae 1898-us-apom	1
1888-hj-tr 1893-haj-dtn	0.556385
1885-fmc-atoalp 1898-fmc-vc	0.745045
1885-fmc-atoalp 1905-fmc-fm	0.536564
1869-espw-mwag 1889-espw-tsfi	0.297986
1891-fmc-do 1891-wta-sbtu	0.590887
1887-fmc-s 1888-haj-bb	0.322945
1893-haj-ianwatgfoa 1894-cdw-tgh	0.350106
1868-espw-tga 1917-hj-tit	0.557853
1883-hj-rh-v2 1904-hj-tgb	0.565194
1898-hj-itc 1908-fmc-tp	0.571067
