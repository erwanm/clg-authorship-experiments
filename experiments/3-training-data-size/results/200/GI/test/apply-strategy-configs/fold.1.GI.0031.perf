0.328130	0.596600	0.550000
