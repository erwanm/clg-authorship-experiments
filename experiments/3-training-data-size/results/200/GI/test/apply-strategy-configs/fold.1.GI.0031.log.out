apply-config.sh: applying config '/tmp/expe3/200/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0031.model/strategy-configs/fold.1.GI.0031' to cases '/home/moreaue/dump/expe3//results/200/GI/test/apply-strategy-configs/only-cases.tmp.pEOrYHxU'; results written to '/home/moreaue/dump/expe3//results/200/GI/test/apply-strategy-configs/fold.1.GI.0031'
apply-config.sh: done.
